src/components/App/App.scss

```diff
@@ -0,0 +1,27 @@
+@use 'variables'
+
+// if i remove this body selector the background goes white
+// even though there are no css rules applied to this selector
+body {
```

Comment:
I pulled down this branch to test some things out, and I was able to comment this out without any changes to the background color.  I'm not sure what is happening on your end but we can take a look at this tomorrow morning.

Reply:
So strange.  Can we compare machines before we merge?  I just added a few more commits with updates to building out the WelcomeForm component and its styling.

Reply:
Absolutely I think that would be a good idea.

Reply:
After comparing our local repos, we've decided to have Colin push his scss corrections to the open PR so that we can merge the corrected version with all changes. 

This will allow us both to pull down all the changes since my local machine is only showing the updates in WelcomeForm, not App for some reason.